### PR TITLE
patch - adding less_than_or_equal and more_than_or_equal to promql alert

### PR DESCRIPTION
**Pull request by OrNovo**
adding less_than_or_equal_usual and more_than_or_equal_usual to metric.promql alert

### Acceptance tests
- [ ] Have you added an acceptance test for the functionality being added?
- [ ] Have you run the acceptance tests on this branch?

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### Release Note
Release note for [CHANGELOG](https://github.com/coralogix/terraform-provider-coralogix/blob/master/CHANGELOG.md):

```release-note
...
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

File: coralogix/resource_coralogix_alert.go [hunk 1]
```diff
@@ -1163,43 +1163,42 @@ func metricSchema() map[string]*schema.Schema {
...
 										" Must be one of - less_than or more_than.",
 								},
 								"more_than": {
 									Type:     schema.TypeBool,
 									Optional: true,
-									ExactlyOneOf: []string{"metric.0.promql.0.condition.0.more_than",
-										"metric.0.promql.0.condition.0.less_than",
-										"metric.0.promql.0.condition.0.more_than_usual",
-										"metric.0.promql.0.condition.0.less_than_usual"},
 									Description: "Determines the condition operator." +
```
Comment (lootag) [inline coralogix/resource_coralogix_alert.go:1179]: Shouldn't the description be updated? As far as I understand the `ExactlyOneOf` block is gone

File: coralogix/resource_coralogix_alert.go [hunk 2]
```diff
@@ -1163,43 +1163,42 @@ func metricSchema() map[string]*schema.Schema {
...
 										" Must be one of - less_than or more_than.",
 								},
 								"more_than_usual": {
 									Type:     schema.TypeBool,
 									Optional: true,
-									ExactlyOneOf: []string{"metric.0.promql.0.condition.0.more_than",
```
Comment (lootag) [inline coralogix/resource_coralogix_alert.go:1186]: Same here

File: coralogix/resource_coralogix_alert.go [hunk 3]
```diff
@@ -1163,43 +1163,42 @@ func metricSchema() map[string]*schema.Schema {
...
-										"metric.0.promql.0.condition.0.less_than",
-										"metric.0.promql.0.condition.0.more_than_usual",
-										"metric.0.promql.0.condition.0.less_than_usual"},
 									Description: "Determines the condition operator." +
 										" Must be one of - immediately, less_than, more_than or more_than_usual.",
 								},
 								"less_than_usual": {
 									Type:     schema.TypeBool,
 									Optional: true,
-									ExactlyOneOf: []string{"metric.0.promql.0.condition.0.more_than",
```
Comment (lootag) [inline coralogix/resource_coralogix_alert.go:1196]: Same here

File: coralogix/resource_coralogix_alert.go [hunk 4]
```diff
@@ -1163,43 +1163,42 @@ func metricSchema() map[string]*schema.Schema {
...
 									Description: "Determines the condition operator." +
 										" Must be one of - immediately, less_than, more_than or more_than_usual.",
 								},
+								"more_than_or_equal": {
+									Type:     schema.TypeBool,
+									Optional: true,
+								},
+								"less_than_or_equal": {
+									Type:     schema.TypeBool,
+									Optional: true,
```
Comment (lootag) [inline coralogix/resource_coralogix_alert.go:1200]: Isn't this missing a description?

File: coralogix/resource_coralogix_alert.go [hunk 5]
```diff
@@ -1163,43 +1163,42 @@ func metricSchema() map[string]*schema.Schema {
...
-									ExactlyOneOf: []string{"metric.0.promql.0.condition.0.more_than",
-										"metric.0.promql.0.condition.0.less_than",
-										"metric.0.promql.0.condition.0.more_than_usual",
-										"metric.0.promql.0.condition.0.less_than_usual"},
 									Description: "Determines the condition operator." +
 										" Must be one of - immediately, less_than, more_than or more_than_usual.",
 								},
+								"more_than_or_equal": {
+									Type:     schema.TypeBool,
+									Optional: true,
```
Comment (lootag) [inline coralogix/resource_coralogix_alert.go:1196]: Isn't this missing a description as well?

File: docs/resources/alert.md [hunk 6]
```diff
@@ -711,18 +711,14 @@ Required:
...
+- `less_than_usual` (Boolean) Determines the condition operator. Must be one of - immediately, less_than, more_than or more_than_usual.
+- `manage_undetected_values` (Block List, Max: 1) Manage your logs undetected values - when relevant, enable/disable triggering on undetected values and change the auto retire interval. By default (when relevant), triggering is enabled with retire-ratio=NEVER. (see [below for nested schema](#nestedblock--metric--promql--condition--manage_undetected_values))
 - `min_non_null_values_percentage` (Number)
 - `more_than` (Boolean) Determines the condition operator. Must be one of - less_than or more_than.
-- `more_than_usual` (Boolean) Determines the condition operator. Must be one of - immediately, less_than, more_than or
-  more_than_usual.
-- `replace_missing_value_with_zero` (Boolean) If set to true, missing data will be considered as 0, otherwise, it will
-  not be considered at all.
+- `more_than_or_equal` (Boolean)
+- `more_than_usual` (Boolean) Determines the condition operator. Must be one of - immediately, less_than, more_than or more_than_usual.
```
Comment (lootag) [inline docs/resources/alert.md:720]: Same in the readme. Shouldn't it be updated?

File: coralogix/resource_coralogix_alert.go (hunk 1 above)
Comment (OrNovo) [reply coralogix/resource_coralogix_alert.go:1179]: The other validations were unnecessary, as ExactlyOneOf already checks that exactly one of those parameters were defined, and it doesn't have to be check for every one of the same parameters.

File: coralogix/resource_coralogix_alert.go (hunk 4 above)
Comment (OrNovo) [reply coralogix/resource_coralogix_alert.go:1202]: fixed

File: coralogix/resource_coralogix_alert.go (hunk 5 above)
Comment (OrNovo) [reply coralogix/resource_coralogix_alert.go:1196]: fixed

File: docs/resources/alert.md (hunk 6 above)
Comment (OrNovo) [reply docs/resources/alert.md]: fixed

**Pull request merged**